mkdocs/themes/mkdocs/keyboard-modal.html

```diff
@@ -0,0 +1,41 @@
+<div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="Keyboard Shortcuts Modal" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+                <h4 class="modal-title" id="exampleModalLabel">Keyboard Shortcuts</h4>
+            </div>
+            <div class="modal-body">
+              <table class="table">
+                <thead>
+                  <tr>
+                    <th style="width: 20%;">Keys</th>
+                    <th>Action</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td><kbd>?</kbd></td>
+                    <td>Open this help</td>
+                  </tr>
+                  <tr>
+                    <td><kbd>←</kbd></td>
+                    <td>Previous page</td>
+                  </tr>
+                  <tr>
+                    <td><kbd>→</kbd></td>
+                    <td>Next page</td>
+                  </tr>
+                  <tr>
+                    <td><kbd>Alt</kbd>+<kbd>s</kbd></td>
```

Comment:
I noticed that the plus sign looks out-of-line (vertically) with the `Alt` and `s` keys. Its a nitpick, but it would be nice to get it right.

Actually, as per [this][1], you should have:

[1]: https://www.w3.org/wiki/HTML/Elements/kbd#Example_B

```html
<td><kbd><kbd>Alt</kbd>+<kbd>s</kbd></kbd></td>
```

That way, the plus sign is still in the outer `kbd` element, but separate from the actual keys in the inner `kdb` elements. Then just apply the key styling to the inner elements. That may resolve the issue itself. If not, then it at least provides an extra styling hook.